text/0000-extend-supported-asset-file-types-svg-webp.md

```diff
@@ -0,0 +1,36 @@
+- Start Date: 2020-07-15
+- Relevant Team(s): Ember CLI
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# Extend the supported asset file types to include svg and webp (& possibly other)
+
+## Summary
+
+Support commonly used image formats svg and webp out of the box.
+
+## Motivation
+
+A default ember app should support commonly used image formats without requiring advanced changes to the config.
+
+## Detailed design
+
+Ember uses [broccoli-asset-rev](https://github.com/rickharrison/broccoli-asset-rev) to generate the appropriate urls within the ember application. The default configuration includes the following file types: `['js', 'css', 'png', 'jpg', 'gif', 'map']`. These days `svg` images are regularly used for vector graphics and `webp` is used in order to save bandwith (where supported). However, if these files are added to a new ember application, they are treated differently than the ones listed above. Fingerprinting is not enabled and related configurations like `prepend` are ignored. This can lead to anything from images not being displayed up to difficult to find issues like images being loaded from the wrong server, or broken cache invalidation.
```

Comment:
While each sentence is clear, I think breaking the long paragraph into a few will help. What do you think about:

``````suggestion
Ember uses [broccoli-asset-rev](https://github.com/rickharrison/broccoli-asset-rev) to generate the appropriate urls within the Ember application. Ember's default value for `extensions` is currently the same as `broccoli-asset-rev`'s:

```javascript
// Current default extensions
['js', 'css', 'png', 'jpg', 'gif', 'map']
```

These days, `svg` images are regularly used for vector graphics and `webp` is used in order to save bandwidth (where supported). If `svg` and `webp` files are added to a new Ember application, they are treated differently than files with extensions listed above. Fingerprinting is not enabled and related configurations like `prepend` are ignored.

This can lead to:

- Images not being displayed
- Images being loaded from the wrong server
- Broken cache invalidation
``````